mmv1/third_party/terraform/services/bigquery/iam_bigquery_dataset.go

```diff
@@ -222,19 +222,19 @@ func iamMemberToAccess(member string) (string, string, error) {
 			return "groupByEmail", pieces[1], nil
 		case "domain":
 			return "domain", pieces[1], nil
+		case "iamMember":
+			return "iamMember", pieces[1], nil
 		case "user":
 			return "userByEmail", pieces[1], nil
 		case "serviceAccount":
 			return "userByEmail", pieces[1], nil
-		default:
-			return "", "", fmt.Errorf("Failed to parse BigQuery Dataset IAM member type: %s", member)
 		}
 	}
 	if member == "projectOwners" || member == "projectReaders" || member == "projectWriters" || member == "allAuthenticatedUsers" {
```

Comment:
Opportunistic refactoring/fix opportunity:
1- projectWriters, projectReaders, projectOwners are not documented as possible output https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_dataset_iam#google_bigquery_dataset_iam_member 

2- allUsers is documented but doesn't look to be a case handled by the API (?). https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets#Dataset

Reply:
Feel free to ignore this comment if not relevant / I lack of context but I just ping you in case you missed this comment @LucaPrete .